src/Uno.XamlMerge.Tests/Scenarios/When_Different_Namespace_Should_Be_Merged_Multiple_Attributes/expected.xml

```diff
@@ -0,0 +1,6 @@
+<ResourceDictionary xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:wasm="http://uno.ui/wasm#using:NS1;NS2" xmlns:android="http://uno.ui/android#using:AndroidNS" xmlns:ios="http://uno.ui/ios#using:iosNS" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+  <!--origin: When_Different_Namespace_Should_Be_Merged_Multiple_Attributes\Input_Dictionary_1.xml-->
+  <MyElement1 x:Name="Name" android:MyProp1="P1 - &#xE735;" ios:MyProp2="P2" />
```

Comment:
This is the validation for escaping @jeromelaban

Reply:
Ah! I missed it, thanks ;)